manticore/__main__.py

```diff
@@ -36,6 +36,11 @@ def positive(value):
                         help=argparse.SUPPRESS)
     parser.add_argument('--env', type=str, nargs=1, default=[], action='append',
                         help='Specify symbolic environment variable VARNAME=++++++')
+    #TODO allow entry as an address
+    #parser.add_argument('--entry', type=str, default=None,
+    #                    help='entry point as an address')
+    parser.add_argument('--entrysymbol', type=str, default=None,
+                        help='entry point as a symbol')
```

Comment:
Think you mean "symbol as entry point"? (ditto for comment above)

Reply:
Ok, sorry for my English